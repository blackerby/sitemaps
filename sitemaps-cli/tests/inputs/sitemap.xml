<?xml version="1.0" encoding="UTF-8"?>

<urlset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd" xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">  <url>
    <loc>https://www.govinfo.gov/bulkdata/PLAW/117/private/PLAW-117pvtl1.xml</loc>
    <lastmod>2023-05-19T14:48:04.488Z</lastmod>
    <changefreq>monthly</changefreq>
    <priority>1.0</priority>
  </url>
  <url>
    <loc>https://www.govinfo.gov/bulkdata/PLAW/117/private/PLAW-117pvtl2.xml</loc>
    <lastmod>2023-05-19T14:48:04.883Z</lastmod>
    <changefreq>monthly</changefreq>
    <priority>1.0</priority>
  </url>
  <url>
    <loc>https://www.govinfo.gov/bulkdata/PLAW/117/private/PLAW-117pvtl3.xml</loc>
    <lastmod>2023-04-17T14:54:05.011Z</lastmod>
    <changefreq>monthly</changefreq>
    <priority>1.0</priority>
  </url>
</urlset>
