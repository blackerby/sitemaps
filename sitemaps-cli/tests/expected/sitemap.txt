loc                                                                  lastmod                   changefreq  priority
https://www.govinfo.gov/bulkdata/PLAW/117/private/PLAW-117pvtl1.xml  2023-05-19T14:48:04.488Z  monthly     1.0
https://www.govinfo.gov/bulkdata/PLAW/117/private/PLAW-117pvtl2.xml  2023-05-19T14:48:04.883Z  monthly     1.0
https://www.govinfo.gov/bulkdata/PLAW/117/private/PLAW-117pvtl3.xml  2023-04-17T14:54:05.011Z  monthly     1.0
